tests/python/gaia-ui-tests/gaiatest/gaia_test.py

```diff
@@ -700,6 +732,7 @@ def unlock(self):
         self.marionette.import_script(self.lockscreen_atom)
         self.marionette.switch_to_frame()
         result = self.marionette.execute_async_script('GaiaLockScreen.unlock()')
+        GaiaData(self.marionette).set_setting('lockscreen.enabled', False)
```

Comment:
I'm not too concerned about this, but I understand what you're saying. For what it's worth, even the atom has side-effects (unlocking also disables the screen timeout), which I've never been too keen on.